PiicoDev_Unified.py

```diff
@@ -141,6 +161,16 @@ def write8(self, addr, reg, data):
     def read16(self, addr, reg):
         regInt = int.from_bytes(reg, 'big')
         return self.i2c.read_word_data(addr, regInt).to_bytes(2, byteorder='little', signed=False)
+    
+    def scan(self): #From Adafruit Blinka generic_linux
+        found = []
+        for addr in range(0, 0x80):
```

Comment:
Redundant 0, could just be range(0x80) (and we're using base-10 and hex in the same range call) <- May be worth checking removing the zero works on all platforms

Reply:
in fact, I2C only has a (simple) valid address pool 0x08 - 0x77 so we ought to implement that within scan instead.
Addresses above 0x77 are either reserved, or used for 10-bit addressing.
Below 0x07 is used for general calls, compatibility and other exotic features.